objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/imageRetriever/NetworkStreamRetriever.java

```diff
@@ -186,8 +191,10 @@ public void diagnoseConnection() throws ImageAcquisitionException{
             URL urlProtocolTest = new URL((String) camera);
             InputStream urlReadTest = urlProtocolTest.openStream();
         } catch (MalformedURLException e) {
-            throw new IllegalArgumentException(this.getClass().getCanonicalName() + ".unknownProtocol: "
-                    + "URL specifies unknown protocol");
+//            throw new IllegalArgumentException(this.getClass().getCanonicalName() + ".unknownProtocol: "
+//                    + "URL specifies unknown protocol");
+            log.error(".unknownProtocol: URL specifies unknown protocol");
```

Comment:
Better would be something akin to

log.error(this.getClass().getCanonicalName() + ".unknownProtocol: Error parsing URL: ", e)

that will include the actual underlying error (e) in the error message